upb/json/parser.rl

```diff
@@ -1290,15 +1290,19 @@ static void start_timestamp_base(upb_json_parser *p, const char *ptr) {
 static bool end_timestamp_base(upb_json_parser *p, const char *ptr) {
   size_t len;
   const char *buf;
+  char timestamp_buf[23];
 
   if (!capture_end(p, ptr)) {
     return false;
   }
 
   buf = accumulate_getptr(p, &len);
+  memset(timestamp_buf, 0, 23);
+  memcpy(timestamp_buf, buf, len);
```

Comment:
please check that len<sizeof(timestamp_buf).

Reply:
Can I use a UPB_TIMESTAMP_BASE_SIZE (19) here?